{"primaryContentSections":[{"kind":"declarations","declarations":[{"tokens":[{"kind":"keyword","text":"var"},{"kind":"text","text":" "},{"kind":"identifier","text":"message"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"String","preciseIdentifier":"s:SS"},{"kind":"text","text":" { "},{"kind":"keyword","text":"get"},{"kind":"text","text":" }"}],"languages":["swift"],"platforms":["iOS"]}]}],"schemaVersion":{"major":0,"minor":3,"patch":0},"sections":[],"variants":[{"paths":["\/documentation\/mgenetwork\/networkerror\/message"],"traits":[{"interfaceLanguage":"swift"}]}],"identifier":{"url":"doc:\/\/MGENetwork\/documentation\/MGENetwork\/NetworkError\/message","interfaceLanguage":"swift"},"abstract":[{"type":"text","text":"A message for better understanding the error."}],"kind":"symbol","metadata":{"fragments":[{"kind":"keyword","text":"var"},{"kind":"text","text":" "},{"kind":"identifier","text":"message"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"String","preciseIdentifier":"s:SS"}],"title":"message","roleHeading":"Instance Property","role":"symbol","symbolKind":"property","externalID":"s:10MGENetwork12NetworkErrorO7messageSSvp","modules":[{"name":"MGENetwork"}]},"hierarchy":{"paths":[["doc:\/\/MGENetwork\/documentation\/MGENetwork","doc:\/\/MGENetwork\/documentation\/MGENetwork\/NetworkError"]]},"references":{"doc://MGENetwork/documentation/MGENetwork/NetworkError":{"role":"symbol","title":"NetworkError","fragments":[{"kind":"keyword","text":"enum"},{"kind":"text","text":" "},{"kind":"identifier","text":"NetworkError"}],"abstract":[{"type":"text","text":"Error occurring upon network tasks."}],"identifier":"doc:\/\/MGENetwork\/documentation\/MGENetwork\/NetworkError","kind":"symbol","type":"topic","navigatorTitle":[{"kind":"identifier","text":"NetworkError"}],"url":"\/documentation\/mgenetwork\/networkerror"},"doc://MGENetwork/documentation/MGENetwork":{"role":"collection","title":"MGENetwork","abstract":[],"identifier":"doc:\/\/MGENetwork\/documentation\/MGENetwork","kind":"symbol","type":"topic","url":"\/documentation\/mgenetwork"},"doc://MGENetwork/documentation/MGENetwork/NetworkError/message":{"role":"symbol","title":"message","fragments":[{"kind":"keyword","text":"var"},{"kind":"text","text":" "},{"kind":"identifier","text":"message"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"String","preciseIdentifier":"s:SS"}],"abstract":[{"type":"text","text":"A message for better understanding the error."}],"identifier":"doc:\/\/MGENetwork\/documentation\/MGENetwork\/NetworkError\/message","kind":"symbol","type":"topic","url":"\/documentation\/mgenetwork\/networkerror\/message"}}}